### PR TITLE
added a rejected category

### PR DESCRIPTION
**Pull request by pyalot**
extensions have to placed in the folder rejected, change their root tag to rejected and change their path to the folder rejected.

The file logo-Rejected.png doesn't exist yet, I don't have the source to the PNG file to generate it.

### CODE REVIEW

File: resources/Khronos-Rejected.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+/* Style for a public "Working Draft" */
+
+@import "default.css";
+
+body {
+  background-image: url('logo-Rejected.png');
```
Comment (kenrussell) [inline resources/Khronos-Rejected.css]: Let's just elide the "rejected" logo. It's not worth asking anybody to draw one up. Perhaps consider adding some other banner to make it clear that this extension is not to be implemented.

**Comment by pyalot** [comment]
Let's just skip the banner for this extension, I think it's reasonably clear that rejected extensions are rejected.

I have also adjusted the text on the registry page to reflect the process of rejecting extensions and to clarify that every extension should move to Khronos ratified.

File: extensions/registry.html [hunk 2]
```diff
@@ -106,9 +106,16 @@ <h2 class="no-toc">Extension Development Process</h2>
     <p>
         Extensions move through four states during their
         development: <em>proposed</em>, <em>draft</em>, <em>community approved</em>, and <em>Khronos
-        ratified</em>.
+        ratified</em>. Every extension should advance to <em>Khronos ratified</em>.
+        If an extension cannot advance trough the extension process it can be <em>rejected</em>.
```
Comment (dsheets) [inline extensions/registry.html]: s/trough/through/

**Comment by pyalot** [comment]
Fixed trough typo and simplified the wording of rejected extensions.

**Comment by kenrussell** [comment]
Florian, thanks for this contribution and David, thanks for reviewing it. Looks good.

**Pull request merged**